### DISCUSSION TITLE
"Borealis" feature

### DISCUSSION THREAD

**Discussion by kam193** [CybercentreCanada/assemblyline#258] [Q&A] [answered]
Hey! 

After the last upgrade, I see "Show Borealis details" link in the context menu on tags. However, the link does nothing. I have found a few references to borealis in the code, corresponding UI package, and even a link to https://github.com/CybercentreCanada/borealis-api (which appears to be private?), but no explanation. Is it a private feature, and the link just shouldn't be visible when not enabled, or I'm just bad at searching?

**Comment by cccs-sgaron** [comment, marked as answer]
Its an upcomming feature that is not released yet that will link AL to another service. It should have been hidden until we are ready to talk about this. I will disable the menu until we are ready to share the details. Good find 😉
